iptables/table.go

```diff
@@ -263,6 +308,10 @@ func NewTable(
 
 		newCmd: newCmd,
 		sleep:  sleep,
+
+		gaugeNumChains:        gaugeNumChains.WithLabelValues(fmt.Sprintf("%d", ipVersion), name),
```

Comment:
Unfortunately not, the other one is already a typedeffed string, I don't think you can stringify a number like that.